### PR TITLE
feat: add directory picker dialog to root folder input

### PR DESCRIPTION
**Pull request by corquaid**
Description
---
- [x] Add Tauri dialog API directory picker to root folder input field
- [x] Fix settings button styling on `BaseNode`  tab

Motivation and Context
---
#343 

How Has This Been Tested?
---

https://user-images.githubusercontent.com/69508715/176632727-e9823ddd-2b1e-4509-a1bc-a922bd3dfa41.mov

With updated `defaultPath` prop:

https://user-images.githubusercontent.com/69508715/176651416-523c6692-cfc3-4c56-ad2f-0a272759be88.mov

### CODE REVIEW

File: applications/launchpad/gui-react/src/containers/SettingsContainer/BaseNodeSettings/BaseNodeSettings.tsx [hunk 1]
```diff
@@ -22,12 +25,26 @@ import MessagesConfig from '../../../config/helpMessagesConfig'
...
   control: Control<SettingsInputs>
   onBaseNodeConnectClick: () => void
+  setValue: UseFormSetValue<SettingsInputs>
 }) => {
   const theme = useTheme()
   const dispatch = useAppDispatch()
+  const selectDirectory = useCallback(async () => {
+    const selectedFolder = await open({
+      directory: true,
+      defaultPath: await appDir(),
```
Comment (tomaszantas) [inline applications/launchpad/gui-react/src/containers/SettingsContainer/BaseNodeSettings/BaseNodeSettings.tsx]: I would suggest I change here:
Now it always opens on the same directory. Even if we already used this dialog to select some directory, that new one selection is not a default path in new interaction. So we can do this:

```suggestion
      const selectDirectory = useCallback(async (lastPath?: string) => {
    const selectedFolder = await open({
      directory: true,
      defaultPath: lastPath || await appDir(),
```

... and then we need to make another change in the input...

File: applications/launchpad/gui-react/src/containers/SettingsContainer/BaseNodeSettings/BaseNodeSettings.tsx [hunk 2]
```diff
@@ -64,6 +81,7 @@ const BaseNodeSettings = ({
           <InputRow>
             <Label $noMargin>{t.baseNode.settings.rootFolder}</Label>
             <Input
+              onClick={() => selectDirectory()}
```
Comment (tomaszantas) [inline applications/launchpad/gui-react/src/containers/SettingsContainer/BaseNodeSettings/BaseNodeSettings.tsx]: ... this is the second change:

```suggestion
              onClick={() => selectDirectory(field.value)}
```

**Review by tomaszantas — CHANGES_REQUESTED** [review]

File: applications/launchpad/gui-react/src/containers/SettingsContainer/BaseNodeSettings/BaseNodeSettings.tsx (hunk 1 above)
Comment (corquaid) [reply applications/launchpad/gui-react/src/containers/SettingsContainer/BaseNodeSettings/BaseNodeSettings.tsx]: Done

File: applications/launchpad/gui-react/src/containers/SettingsContainer/BaseNodeSettings/BaseNodeSettings.tsx (hunk 2 above)
Comment (corquaid) [reply applications/launchpad/gui-react/src/containers/SettingsContainer/BaseNodeSettings/BaseNodeSettings.tsx]: And done.

**Review by tomaszantas — APPROVED** [review]

**Review by tarnas14 — APPROVED** [review]

**Pull request merged**